SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,30 +1,25 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
-
 # Singleton Actors
 
 Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. They are server-side authoritative [Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS.
```

Comment:
I think we can just say "Unreal Actors" here to make it clear it's not any funny business.
So: 
"They are server-side authoritative Unreal Actors that..."